src/frontend/pages/general/SignIn/components/SignInBox.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import MUIButtonBase from '@material-ui/core/ButtonBase';
+import MUITypography from '@material-ui/core/Typography';
+import GoogleSVG from '../assets/GoogleIcon.svg';
```

Comment:
Wondering if it makes more sense for these files to be under `/frontend/components`, or `frontend/pages/general/SignIn/components`.

Reply:
I think since it's not a component reused anywhere else it's okay for it to be in the current folder directory. I recall that was the structure we wanted to follow for CMS.